src/relayFeeCalculator/relayFeeCalculator.ts

```diff
@@ -50,7 +60,52 @@ export class RelayFeeCalculator {
       this.capitalCostsPercent >= 0 && this.capitalCostsPercent <= 100,
       "capitalCostsPercent must be between 0 and 100 percent"
     );
+    this.capitalCostsConfig = {};
```

Comment:
i would highly suggest letting the caller of the class do the parsing of the json, and have types defined for these config parameters, rather than passing a json in as a string. that way they can use intellisense to discover what the config fields should be.  Its also a bit wierd from a UX perspective to pass stringified json into a class constructor.